src/app/[locale]/specs/sync/page.mdx

```diff
@@ -0,0 +1,171 @@
+export const metadata = {
+  title: 'Sync',
+  description:
+    'Firehose and other data synchronization mechanisms.',
+}
+
+# Data Synchronization
+
+One of the main design goals of atproto (the "Authenticated Transfer Protocol") is to reliably distribute public content between independent network services. This data transfer should be trustworthy (cryptographicly authenticated) and relatively low-latency even at large scale. It is also important that new participants can join the network at any time and “backfill” prior content.
+
+This section describes the major data synchronization features in atproto. The primary real-time data synchronization mechanism is repository event streams, commonly referred to as "firehoses". The primary batch data transfer mechanism is repository exports as CAR files. These two mechanisms can be combined in a "bootstrapping" process which result in a live-synchronized copy of the network.
+
+## Synchronization Primitives
+
+As described in the repository spec, each commit to a repository has a *revision* number, in TID syntax. The revision number for an account must always increase, even if the account is migrated between hosts, or has a period of inactivation in the network. To make this simple, it is recommended to just use the current time as the TID, even when starting with a new account. Services should reject or ignore revision numbers corresponding to future timestamps (beyond a short fuzzy time drift window). Network services can track the commit revision for every account they have seen, and use this to verify synchronization progress. Services which synchronize data can include the most-recently-processed revision in HTTP responses to API requests from the account in question, in the `Atproto-Repo-Rev` HTTP response header. This allows clients (and users) to detect if the response is up-to-date with the actual repository, and detect any problems with synchronization.
```

Comment:
```suggestion
As described in the repository spec, each commit to a repository has a *revision* number, in TID syntax. The revision number for successive commits must always increase, even if the account is migrated between hosts, or has a period of inactivation in the network. The revision number of the most recent commit can serve as a counter to guide synchronizations. To make this simple, it is recommended to just use the current time as the TID for each commit, even for the initial commit when creating a new account. Services should reject or ignore revision numbers corresponding to future timestamps (beyond a short fuzzy time drift window). Network services can track the commit revision for every account they have seen, and use this to verify synchronization progress. Services which synchronize data can include the most-recently-processed revision in HTTP responses to API requests from the account in question, in the `Atproto-Repo-Rev` HTTP response header. This allows clients (and users) to detect if the response is up-to-date with the actual repository, and detect any problems with synchronization.
```

Reply:
also what's with `inactivation`? is that a deliberate portmanteau of 'inactivity' and 'deactivation'? 😉

Reply:
"inactivity"